platform/darwin/src/MGLGeoJSONSourceBase_Private.h

```diff
@@ -0,0 +1,13 @@
+#import "MGLGeoJSONSource.h"
+
+#include <mbgl/style/sources/geojson_source.hpp>
+
+@interface MGLGeoJSONSourceBase (Private)
+
+@property (nonatomic, readwrite) NSDictionary *options;
+
+- (instancetype)initWithIdentifier:(NSString *)identifier options:(NS_DICTIONARY_OF(NSString *, id) *)options;
```

Comment:
Should this initializer be public (with a warning not to call it directly), as with `-[MGLSource initWithIdentifier:]`?